cartoframes/data/clients/bigquery_client.py

```diff
@@ -115,7 +114,7 @@ def download_to_dataframe(self, job):
                 if job.errors:
                     log.error([error['message'] for error in job.errors if 'message' in error])
 
-                raise CartoException('Error downloading data')
+                raise Exception('Error downloading data')
```

Comment:
Same as above with more specific error. On this one, should we carry through the error from BQ?